toolkits/global/packages/global-javascript/README.md

```diff
@@ -20,10 +20,11 @@ You can import as many of the named exports from the helpers as you require for
 - [getCookie](#getcookie)
 - [getCookie](#debounce)
 - [getCookie](#throttle)
```

Comment:
Not part of this PR, but the name are wrong on these!
```
[getCookie](#debounce)
[getCookie](#throttle)
```

Should be:

```
[debounce](#debounce)
[throttle](#throttle)
```

Reply:
😬